tools/Migrate_01/OldTriggers/ContentChangedTriggerSchema.cs

```diff
@@ -50,19 +50,9 @@ public ContentChangedTriggerSchemaV2 Migrate()
                 conditions.Add($"event.type == '{EnrichedContentEventType.Published}'");
             }
 
-            if (SendUnpublish)
-            {
-                conditions.Add($"event.type == '{EnrichedContentEventType.Unpublished}'");
-            }
-
             if (SendArchived)
             {
-                conditions.Add($"event.type == '{EnrichedContentEventType.Archived}'");
-            }
-
-            if (SendRestore)
-            {
-                conditions.Add($"event.type == '{EnrichedContentEventType.Restored}'");
+                conditions.Add($"event.status == 'Archived'");
```

Comment:
Is this correct?

Reply:
No, should be 'Draft', thx

Reply:
No,  everything is fine. The Change here looks just a little bit strange.